ui/hooks/src/use-invite-token-validation.ts

```diff
@@ -20,6 +20,12 @@ const checkIsValidToken = async (inviteToken: string) => {
 
 /**
  * Hook to validate invitation token
+ * @example useIsValidToken hook
+ * ```typescript
+ * const inviteTokenQuery = useIsValidToken({ inviteToken: '5b71c9dd7bdea84c9ff2564a', enabler: true });
```

Comment:
5b71c9dd7bdea84c9ff2564a

Reply:
That is a valid invite token format, tho already used

Reply:
then use `received-invite-token`